game.py

```diff
@@ -75,12 +83,50 @@ def check_events(self):
                 if event.key == pygame.K_RIGHT:
                     self.RIGHT_KEY = False
 
+    def asteroids_update(self):
```

Comment:
This method is probably better placed in the `Asteroid` class. You have also access to the game instance in it, since it's a parameter in the constructor.

Reply:
OK, I've moved it to the `Asteroid` class. It's not just dealing with asteroids since it is also updating the rocket life, but I agree that it makes more sense to take it out of the `game` class.